plugins/source/aws/resources/services/backup/jobs.go

```diff
@@ -0,0 +1,44 @@
+package backup
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+	"github.com/aws/aws-sdk-go-v2/service/backup/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Jobs() *schema.Table {
+	tableName := "aws_backup_jobs"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/aws-backup/latest/devguide/API_BackupJob.html`,
+		Resolver:    fetchBackupJobs,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "backup"),
+		Transform:   transformers.TransformWithStruct(&types.BackupJob{}, transformers.WithPrimaryKeys("BackupJobId")),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(true),
+			client.DefaultRegionColumn(true),
+		},
+	}
+}
+
+func fetchBackupJobs(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Backup
+	params := backup.ListBackupJobsInput{ByAccountId: aws.String(cl.AccountID), MaxResults: aws.Int32(1000)} // maximum value from https://docs.aws.amazon.com/aws-backup/latest/devguide/API_ListBackupJobs.html
```

Comment:
`ByAccountId` can be `*` to list all jobs for the account, but that doesn't sit well with our multiplexing.
It seems better to multiplex and save the account and region data